readme.md

```diff
@@ -239,6 +239,22 @@ Show output from DTrace or perf(1) tools.
 
 Default: false
 
+### --source-maps
+
+If a javascript library has a source map (a sourceMappingUrl comment on the last
+line), list the source file and position instead of the original file.
+
+A path to a json file can be supplied to define source maps which can not be
+resolved automatically. The JSON file should list an object with library name as
+key and the source map file as value.
+
+### --relative-path
```

Comment:
Is this really necessary? The source map support should also be possible without that, right?
AFAIK `output-dir` should already be enough in this case.

Reply:
Not sure what you mean, but the idea was that you can both use `--source-maps` as `-source-maps <file>`. In the first case source maps are resolved either by the sourceMapUrl comment or because a file ending on `.map` exists (eg `dist.js.map` for `dist.js`).
In the latter you can specify which source map to use in case a library doesnt specify one. Not sure if this will ever be the case, but as the name uses `url` this could also potentially mean (I guess) a library points to a remote source map on a webserver. If that would be the case, then you could specify it manually.